.idea/modules.xml

```diff
@@ -3,8 +3,11 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/HintedTextView.iml" filepath="$PROJECT_DIR$/HintedTextView.iml" />
+      <module fileurl="file://$PROJECT_DIR$/HintedTextView.iml" filepath="$PROJECT_DIR$/HintedTextView.iml" />
```

Comment:
Yes, I added `.idea/` to `.gitignore` but it seems that I accidentally added it to the commit before :) You can ignore this or delete if you want to